pkg/plugins/platform/local_test.go

```diff
@@ -131,6 +131,24 @@ func TestLocalStart(t *testing.T) {
 
 		cleanup()
 	}
+
+	// try to start a site that is already running
+	err = app.Init(TestSites[0].Dir)
+	assert.NoError(err)
+
+	err = app.Start()
+	assert.EqualError(err, fmt.Sprintf("a site with the name %s is already running", TestSites[0].Name))
```

Comment:
I might not be understanding fully. Just to make sure, let me rephase my first comment a different way. If I had two different "wordpress" repos and I started (1) and was about to start (2), this would be a proper warning. The issue from an end-user would be "ok, so how do I fix that?", which is why I was suggesting a hint about how to resolve by re-running ddev config.